### PR TITLE
13 feature 7 game artworks

### PR DESCRIPTION
**Pull request by Kevin22888**
The card images are done! Let me know if they look cool :D

### CODE REVIEW

File: src/main/java/interface_adapters/CardImageMapper.java [hunk 1]
```diff
@@ -1,4 +1,47 @@
...
+            ImageIcon imageIcon = new ImageIcon(imageFile.getAbsolutePath());
+
+            // Remove the .png extension
+            String name = imageName.substring(0, imageName.length() - 4);
+            // Make the name match those in the JSON files
+            name = name.replace('-', ' ');
+            // This is an awkward case due to our name choices, for now we just leave it here
+            if (name.equals("Demons Amulet")) {
+                name = "Demon's Amulet";
+            }
```
Comment (ethanhsu8004) [inline src/main/java/interface_adapters/CardImageMapper.java:39]: Couldn't we just change the name and delete this code?

File: src/main/java/interface_adapters/CardImageMapper.java [hunk 2]
```diff
@@ -1,4 +1,47 @@
...
+        }
+
+        for (File imageFile : imageFiles) {
+            String imageName = imageFile.getName();
+            ImageIcon imageIcon = new ImageIcon(imageFile.getAbsolutePath());
+
+            // Remove the .png extension
+            String name = imageName.substring(0, imageName.length() - 4);
+            // Make the name match those in the JSON files
+            name = name.replace('-', ' ');
```
[the remark this answers is not in the document]
Comment (Kevin22888) [reply src/main/java/interface_adapters/CardImageMapper.java:35]: Yeah for now there aren't many cards so I'm just running this line every time

File: src/main/java/interface_adapters/CardImageMapper.java (hunk 1 above)
Comment (Kevin22888) [reply src/main/java/interface_adapters/CardImageMapper.java:39]: Haha yeah I guess we can change the name entirely, we'll do that later, for now this doesn't cause big problems

**Review by ethanhsu8004 — APPROVED** [review]
images look good!

**Pull request merged**